wp-content/themes/chrisdinh-photography/resources/scripts/util/filterGallery.js

```diff
@@ -0,0 +1,53 @@
+import { gsap } from "gsap";
+import { Flip } from "gsap/dist/Flip.js";
+
+// https://gsap.com/community/forums/topic/30406-animate-growing-grid-items/
+// https://codepen.io/GreenSock/pen/mdPzJKp
+
+gsap.registerPlugin(Flip);
+
+const filterGallery = () => {
+  const allFilter = document.querySelector('button.gallery-collections__collection--all');
+  const filters = gsap.utils.toArray('.gallery-collections__collection:not(.gallery-collections__collection--all)');
+  const items = gsap.utils.toArray('.gallery-item__image');
+
+  if ( filters.length ) {
+    filters.forEach(btn => btn.addEventListener('click', () => {
+      updateFilters(filters, items, false);
+    }));
+  }
+
+  if ( allFilter ) {
+    allFilter.addEventListener('click', () => {
+      updateFilters(filters, items, true);
+    });
+  }
+}
```

Comment:
Both `filters` and `allFilter` are wrapped within a conditional to ensure either `filters` has a length greater than 0 and `allFilter` is not undefined so thats already being done unless i'm missing something? @jonathanmdinh 

This also may be stale since we're using nanoGallery to handle the grid and filtering - i'll have to double check and see. I may remove this file in a later commit - will comment here if i do

Reply:
Ah you're right, thanks for the clarification. I also figured it was stale due to nanogallery being implemented.